src/components/AddressSearch/index.js

```diff
@@ -342,6 +364,26 @@ function AddressSearch(props) {
         };
     }, []);
 
+    const listEmptyComponent = useCallback(
+        () =>
+            network.isOffline || !isTyping ? null : (
+                <Text style={[styles.textLabel, styles.colorMuted, styles.pv4, styles.ph3, styles.overflowAuto]}>{translate('common.noResultsFound')}</Text>
+            ),
+        [isTyping, translate, network.isOffline],
+    );
+
+    const listLoader = useCallback(
```

Comment:
same here

Reply:
It needs to be a function according to the types of `GooglePlacesAutocomplete`